test/python/tensorflow/tf_unittest_runner.py

```diff
@@ -243,10 +244,10 @@ def list_tests(module_list, regex_input):
                 alltests.append(aTestCase.id())
 
     for aTestCaseID in alltests:
```

Comment:
Can you please follow the style here: https://google.github.io/styleguide/pyguide.html?